.github/workflows/build_riscv.yml

```diff
@@ -45,24 +47,21 @@ jobs:
             debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev \
             pylint xterm zstd liblz4-tool
           sudo apt-get clean
-      
+
       # checkout repository
       - name: Clone meta-riscv, openembedded-core, bitbake, meta-openembedded
         run: |
-          sudo chown runner /mnt
           git clone  https://github.com/riscv/meta-riscv.git
           git clone -b ${TARGET_VERSION} https://github.com/openembedded/openembedded-core.git
           git clone -b ${TARGET_BITBAKE_VERSION} https://github.com/openembedded/bitbake.git
           git clone -b ${TARGET_VERSION} https://github.com/openembedded/meta-openembedded.git
-        working-directory: /home/runner/work
 
       # Run bitbake
       - name: Bitbake MACHINE=${{ matrix.target_machine }}, TensorFlow-Lite version ${{ matrix.onnxruntime_version }}
         run: |
-          source /home/runner/work/openembedded-core/oe-init-build-env build
-          bitbake-layers add-layer /home/runner/work/meta-openembedded/meta-oe
-          bitbake-layers add-layer /home/runner/work/meta-openembedded/meta-python
-          bitbake-layers add-layer /home/runner/work/meta-onnxruntime/meta-onnxruntime/
-          bitbake-layers add-layer /home/runner/work/meta-riscv
+          source openembedded-core/oe-init-build-env build
+          bitbake-layers add-layer ../meta-openembedded/meta-oe
+          bitbake-layers add-layer ../meta-openembedded/meta-python
+          bitbake-layers add-layer ../meta-onnxruntime/
+          bitbake-layers add-layer ../meta-riscv
           MACHINE=${{matrix.target_machine}} bitbake onnxruntime onnxruntime-python-label-image-example
-        working-directory: /mnt
```

Comment:
The reason I chose `/mnt` for the working-directory is because there was 66G of free disk space.
`/` only had about 21G of free space, so I was concerned that even if I deleted unnecessary packages, I wouldn't have enough space.
This was the result of trial and error with [meta-tensorflow-lite](https://github.com/NobuoTsukamoto/meta-tensorflow-lite).
```
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        73G   52G   21G  72% /
tmpfs           7.9G  172K  7.9G   1% /dev/shm
tmpfs           3.2G  1.1M  3.2G   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
/dev/sdb15      105M  6.1M   99M   6% /boot/efi
/dev/sda1        74G  4.1G   66G   6% /mnt
tmpfs           1.6G   12K  1.6G   1% /run/user/1001
```

For information about the free space available for GitHub Actions, please see here (in Japanese).
- https://zenn.dev/pinto0309/scraps/c6413eb15a1b2a

Reply:
The qemuriscv64 log also confirms that there is insufficient disk space.
> No new tasks can be executed since the disk space monitor action is "STOPTASKS"!

https://github.com/NobuoTsukamoto/meta-onnxruntime/actions/runs/9535084300/job/26280377065?pr=7#step:5:4790

Reply:
Thanks, you're right I have removed the last commit and only kept the bump to avoid warning about node16